client/webserver/site/src/css/market_dark.scss

```diff
@@ -24,6 +24,11 @@ body.dark {
     }
   }
 
+  div.numorders {
+    background-color: rgb(25, 25, 180);
```

Comment:
Let's darken this up quite a bit. Maybe like a `#141488`.

Reply:
Maybe we could test out one of our primary colors: https://decred.org/brand/

Keep the hue / rgb ratio and tweak the brightness?  If that clashes with the rest of the DEX scheme, no biggie.

Reply:
![Screen Shot 2021-07-08 at 12 28 08 PM](https://user-images.githubusercontent.com/6186350/124958555-4819f700-dfe8-11eb-84bc-8f2e7564c098.png)
![Screen Shot 2021-07-08 at 12 28 21 PM](https://user-images.githubusercontent.com/6186350/124958563-4c461480-dfe8-11eb-920f-39bb9379a181.png)

The first one is a lighter version of the dark blue primary color and the second is @buck54321 's suggestion. Very similar, but I think @buck54321 's suggestion is a closer match to the dex logo, which doesn't seem to be using any of the Decred colors.

Reply:
k, no sweat.  Just wanted to try to keep us on-brand if possible, but best to stay internally consistent with other dex styles unless we rework the whole scheme.